src/components/Assets/AssetWarrantyCard.tsx

```
@@ -78,20 +78,22 @@
               ["Phone", asset.support_phone, "l-phone"],
               ["Email", asset.support_email, "l-envelope"],
             ].map((item) => (
-              <div className="flex items-center">
+              <div className="flex items-center flex-wrap responsive-width leading-5 mt-2">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)